_rest_config.php

```diff
@@ -289,6 +287,29 @@ public static function authorization_check($section, $value): void
         }
     }
 
+    /**
+     * Checks to make sure the resource the user agent is requesting actually has the right scopes for the api request
+     * scopes are compared against the user's AccessToken scopes.
+     * @param HttpRestRequest $request
+     */
+    public static function scope_check_request(HttpRestRequest $request)
+    {
+        $scopeType = $request->isPatientRequest() ? "patient" : "user";
+        $permission = $request->getRequestMethod() == "GET" ? "write" : "read";
+        $resource = $request->getResource();
+
+        if (!$request->isFhir()) {
+            $resource = strtolower($resource);
```

Comment:
@adunsulag ,
I'm doing some work off the code in this PR to extend your scope checking mechanism to entire api. Is there anything specific that requires the strtolower? (I couldn't figure it out)

Reply:
disregard above question. turns out the real function for scope checking is at checkSecurity, so just ignore my post above.

Reply:
My bad, I should probably remove this function, originally I was using it, but I centralized it all into the dispatch.  I had the strtolower because I noticed all of the resource names in the standard API were lowercased while the FHIR api started with uppercase.